spec/actions_spec.rb

```diff
@@ -232,12 +253,113 @@ def self.name
     end
 
     it 'deletes a non-existing record' do
-      instance = klass.new(value: 1)
+      instance = klass.new(value: 1, indexed_value: 1)
       expect(instance.id).to be_nil
       expect {
         instance.destroy
       }.to_not raise_error
       expect(klass.count).to eq 0
     end
   end
+
+  context 'custom indexes: ' do
```

Comment:
let's move this to index_spec / custom_index_spec?

Reply:
done